packages/devtools_app/test/vm_developer/object_inspector/object_viewport_test.dart

```diff
@@ -253,3 +262,29 @@ void main() {
     });
   });
 }
+
+// testWidgets('test for Field Object', (WidgetTester tester) async {
```

Comment:
are these tests supposed to be uncommented? or removed?

Reply:
Sorry I forgot to delete them 😅